core/field.ts

```diff
@@ -1263,6 +1264,16 @@ export abstract class Field<T = any>
     return false;
   }
 
+  /**
+   * Get the variable model if this field refers to one.
+   *
+   * @returns The field's variable, or null if none exists.
+   * @internal
```

Comment:
If you want this to be overriden in subclasses, it should be public not internal.

Reply:
Thanks! My bad, coming from Python :) 

I was following this pattern from `referencesVariables`, which is `@internal` in `Field` and overridden by `FieldVariable`, also as `@internal`. I suppose we should drop it from there too, then.